content/docs/parsers/opentelemetry/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Parse OpenTelemetry messages
+linktitle: OpenTelemetry
+---
+
+By default, {{% param "product_name" %}} doesn't parse the fields of incoming OpenTelemetry messages into name-value pairs, but are only available for forwarding using the `opentelemetry()` destination. To parse the fields into name-value pairs, use the `opentelemetry()`.
+
+The `opentelemetry()` parser parses the fields into name-value pairs starting with the `.otel.` prefix.
```

Comment:
We could link to the `.proto` files for more info about the parsed fields: https://github.com/open-telemetry/opentelemetry-proto/tree/v0.20.0/opentelemetry/proto